src/result_collection.jl

```diff
@@ -75,15 +77,29 @@ function collect_results!(filename, folder;
     subfolders = false,
     rpath = nothing,
     verbose = true,
+    update = false,
     newfile = false, # keyword only for defining collect_results without !
     kwargs...)
 
     if newfile || !isfile(filename)
         !newfile && verbose && @info "Starting a new result collection..."
         df = DataFrames.DataFrame()
+        mtime_df = nothing
+        mtimes = Dict{String,Float64}()
     else
         verbose && @info "Loading existing result collection..."
-        df = wload(filename)["df"]
+        data = wload(filename)
+        df = data["df"]
+        # Check if we have pre-recorded mtimes (if not this could be because of an old results database).
+        if "mtime" ∈ keys(data)
+            mtimes = data["mtime"]
+            mtime_df = nothing
+        else
+            @warn "Update of existing results collection requested, but no previously recorded modification time found. Will proceed by comparing with the database modification time, but this may miss modifications if the database is newer than the modified files."
```

Comment:
This update warning appears to be thrown even if `update = false` which appears to contradict the statement in the warning.

Reply:
Oops, that just needs a test for `update`.